ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -426,6 +426,37 @@ As of AVD 5.0.0 "null" option for esp integrity and encryption has been replaced
 +      encryption: disabled
 ```
 
+### `router_ospf.redistribute.x.enabled` is required
+
+In AVD 4.0.0, setting `enabled: true` under `router_ospf.redistribute.x` was not
```

Comment:
```suggestion
In AVD 4.0.0, setting `enabled: true` under `router_ospf.redistribute.<x>` was not
```

Reply:
I think mentioning the specific key names here would be better  - `router_ospf.redistribute.connected`, `router_ospf.redistribute.bgp`, `router_ospf.redistribute.static`

Reply:
same for the example, please update the `x` with the exact key names

Reply:
Fixed!